backend/tracim_backend/command/devtools.py

```diff
@@ -185,3 +187,23 @@ def take_app_action(self, parsed_args: argparse.Namespace, app_context: AppEnvir
             "user_{}".format(parsed_args.user_id), message_as_dict=test_message
         )
         print("test message (id=-1) send to user {}".format(parsed_args.user_id))
+
+
+class CustomPropertiesTranslateTemplateCommand(AppContextCommand):
+    """
+    Tool to
+    """
+
+    def get_description(self) -> str:
+        return "create translation template for user custom properties"
+
+    def take_app_action(self, parsed_args: argparse.Namespace, app_context: AppEnvironment) -> None:
+        # TODO - G.M - 05-04-2018 -Refactor this in order
+        # to not setup object var outside of __init__ .
+        self._session = app_context["request"].dbsession
+        self._app_config = app_context["registry"].settings["CFG"]  # type: CFG
+        custom_properties_api = UserCustomPropertiesApi(
+            current_user=None, app_config=self._app_config, session=self._session
+        )
+        # TODO - G.M - 2019-01-12 - Return valid json here (using external library ?)
```

Comment:
yes, pprint can return not valid json as it return `'`  instead of `"` , i decided to not add an external library to return properly json as this is not required for this PR (i just add this small dev tool to help me create examples of translations) and i though it's better to think about it before having a real tool to handle translate like babel is able to do. I do think, the best solution is probably follow similar logic, but this need some work and it's really far from necessary no.